indico/modules/events/surveys/fields/choices.py

```diff
@@ -101,6 +101,12 @@ def render_answer(self, answer):
                             for option_dict in self.question.field_data['options']}
         return question_options.get(answer) or ''
 
+    def copy_field_data(self):
```

Comment:
Since you have this exact same code twice, would it make sense to add a `RemoveOptionIDMixin` that just contains t his method?